BREAKING_CHANGES.md

```diff
@@ -0,0 +1,27 @@
+# Breaking changes in EventFlow
+
+**THIS IS A DRAFT**
+
+This document describes any breaking changes in EventFlow, starting
+from version 0.x to 1.x.
+
+## 1.0
+
+* **No .NET Framework support**
+  * **Reason:** With the announcement of .NET 5, Microsoft stated that
+    .NET Framework is basically legacy, or will be in the very near future.
+    This, and the fact that maintaining boot .NET (Core) and .NET Framework
```

Comment:
boot->both

Reply:
just hacking away to see how far I can get before hitting any blockers